packages/dx-react-grid/docs/reference/custom-summary.md

```diff
@@ -0,0 +1,39 @@
+# CustomSummary Plugin Reference
+
+A plugin that provides customly calculated summary values.
+
+## Importing
+
+Use the following import statement:
+
+```js
+import { CustomSummary } from '@devexpress/dx-react-grid';
+```
+
+## User Reference
+
+### Dependencies
+
+- [SummaryState](summary-state.md)
+
+### Properties
+
+Name | Type | Default | Description
+-----|------|---------|------------
+totalValues | Array&lt;any&gt; | Summary values related to all rows.
+groupValues | { [key: [GroupKey](grouping-state.md#groupkey)]: Array&lt;any&gt; } | Summary values related to group rows.
+treeValues | { [key: number &#124; string]: Array&lt;any&gt; } | Summary values related to rows that contains children rows.
```

Comment:
`Description` section stay in the `Default`
![image](https://user-images.githubusercontent.com/15093892/43580251-7191c842-965d-11e8-826b-168307d58136.png)